.github/workflows/pkgci_regression_test_cpu.yml

```diff
@@ -122,7 +122,27 @@ jobs:
       - name: Run external tests - Models with real weights
         run: |
           source ${VENV_DIR}/bin/activate
-          pytest SHARK-TestSuite/iree_tests/pytorch/models -k real_weights \
-               -rpfE --timeout=600 --durations=0 \
-              --config-files=build_tools/pkgci/external_test_suite/pytorch_models_cpu_llvm_task.json \
-              --no-skip-tests-missing-files
+          pytest SHARK-TestSuite/iree_tests/pytorch/models \
+            -n 4 \
+            -rpfE \
+            -k real_weights \
+            --no-skip-tests-missing-files \
+            --capture=no \
+            --timeout=1200 \
+            --retries 2 \
+            --retry-delay 5 \
+            --durations=0 \
```

Comment:
As we're repeating flags in these workflows, we could move some of them to config files: https://docs.pytest.org/en/stable/reference/customize.html